tests/integration/targets/vmware_host_iscsi/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+# Test code for the vmware_host_logbundle module.
```

Comment:
`vmware_host_logbundle` -> `vmware_host_iscsi`, but I don't think need the header anyway since it's error prone.

Reply:
I fixed the header, should I remove it?

Reply:
Nah it's fine.